### DISCUSSION TITLE
Snapshots and GIFs are all black on Quest

### DISCUSSION THREAD

**Discussion by mikepelletier** [icosa-gallery/open-brush#385] [Q&A] [answered]
I don't seem to be able to make a snapshot or a gif on the quest, the saved images are all black.

**Comment by mikeskydev** [comment, marked as answer]
Hi, I can reproduce this, will nail down some consistent steps to get it logged and fixed. Thanks for reporting!

**Comment by mikeskydev** [comment]
Tracked in #388
